match/lib/match/utils.rb

```diff
@@ -54,6 +51,8 @@ def self.get_cert_info(cer_certificate_path)
       return out_array.map { |x| x.split(/=+/) if x.include?("=") }
                       .compact
                       .map { |k, v| [openssl_keys_to_readable_keys.fetch(k, k), v] }
+                      .append([openssl_keys_to_readable_keys.fetch("notBefore"), cert.not_before])
+                      .append([openssl_keys_to_readable_keys.fetch("notAfter"), cert.not_before])
```

Comment:
hmmmmm, souldn't be this be: `.append([openssl_keys_to_readable_keys.fetch("notAfter"), cert.not_after])`?

Reply:
😱 I'm blind. Fixed.